2021/OffSeason2021/src/main/deploy/2135_configuration.txt

```diff
@@ -2,6 +2,7 @@
   RB_Name = 2135-Grogu
 # DriveTrain settings
   DT_DriveMode = 0
+  DT_StoppedTolerance = 0.05

```

Comment:
What is DT_StoppedTolerance for?

Reply:
It’s how close the actual wheel speeds have to be to zero at the end of the path. Basically it lets us stop path following when the path is complete, since all our paths have zero velocity at the end